packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -251,6 +251,25 @@ export class CalloutContentBase extends BaseComponent<ICalloutProps, ICalloutSta
     }
   }
 
+  // We implement a blur listener for the Callout component so that if a page inside inside of an iframe uses a callout,
+  // it can be dismissed when the user clicks outside of the iframe.
+  protected _dismissOnBlur(ev: Event) {
+    const target = ev.target as HTMLElement;
+    const clickedOutsideCallout = this._hostElement.current && !elementContains(this._hostElement.current, target);
+    const { preventDismissOnLostFocus } = this.props;
+
+    if (
+      !preventDismissOnLostFocus &&
+      ((!this._target && clickedOutsideCallout) ||
```

Comment:
I think you can narrow this and only do 
```
if(!preventDismissOnLostFocus && ev.target === this._targetWindow) { 
    this.dismiss(ev)
} 
```
Since that's the only case we're really trying to capture.